### DISCUSSION TITLE
Image size doesn't change after optimization

### DISCUSSION THREAD

**Discussion by farhadkb6868** [spatie/laravel-image-optimizer#104] [Q&A]
This package does not work on laravel 8.
It didn't raise any error, but the image size does not change after optimization
What is the solution?

**Comment by HuyPham55** [comment]
Facing the same problem. The image seems not to be optimized any at all. 
What I have done:

1. Install the package via composer
2. Publish the vendor assets
3. Use int where needed.

I use the following command to check if ImageOptimizer is really accepting the input image and outputting final cimpressed image. When I compare them to each other, the size (kB) is the same. 
`ImageOptimizer::optimize(base_path().'\public'.'some_user_image.jpg, '/compressed1.jpg');`

**Comment by glitchcodes** [comment]
Check if you have the binaries installed on your environment `jpegoptim, pngquant, or optipng`. If they are not installed, the package will return the same image
